packages/nebula/CHANGELOG.md

```diff
@@ -0,0 +1 @@
+# @thirdweb-dev/nebula
```

Comment:
you should not write the changlog manually, changesets will write it for you, you can remove this file

Reply:
file deleted in favor of changesets